python/pycylon/pycylon/data/table.pxd

```diff
@@ -125,6 +125,19 @@ cdef extern from "../../../../cpp/src/cylon/table.hpp" namespace "cylon":
 
     CStatus Repartition(const shared_ptr[CTable] & table, shared_ptr[CTable] * output)
 
+    CStatus Slice(shared_ptr[CTable] & table,
```

Comment:
Just curious, are we changing the `table` inside the function? If not please use `const`

Reply:
It's a function definition inside cython to communicate with cpp layer. "table" is the input table to be sliced